h/data_tasks/report/create_from_scratch/01_functions/01_date_functions.sql

```diff
@@ -18,6 +18,9 @@ AS $$
                 ELSE INTERVAL '6 months'
             END
         ))::DATE
+        WHEN resolution = 'week' THEN 
+             -- -1 day to start weeks on Sunday
+             (DATE_TRUNC('week', date) - '1 day'::interval)::date
```

Comment:
Change to the function declaration so all usage of this use the same `week` definition.